features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.infer.json

```diff
@@ -46,7 +46,7 @@
 
   "oauth.hash_tokens_and_secrets": {
     "true": {
-      "oauth.extensions.token_persistance_processor": "org.wso2.carbon.identity.oauth.tokenprocessor.EncryptionDecryptionPersistenceProcessor"
+      "oauth.extensions.token_persistance_processor": "org.wso2.carbon.identity.oauth.tokenprocessor.HashingPersistenceProcessor"
```

Comment:
Why is "EncryptionDecryptionPersistenceProcessor" removed?
Check the documentation[1]
[1] - https://docs.wso2.com/display/IS580/Extension+Points+for+OAuth

Reply:
Since hash_tokens_and_secrets is related to only hashing it was removed.

Reply:
Shall we introduce new one for encryption also?

Reply:
Will change to 
Key:oauth.token_storage_method
Values:encrypted/hashed/plaintext

Reply:
changed